src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -1188,6 +1188,25 @@ public virtual ScriptBlock ScriptBlock
 
         private ScriptBlock _scriptBlock;
 
+        /// <summary>
+        /// Initial location of the script block.
+        /// This is used to set the location prior to executing the script block
+        /// </summary>
+        public virtual string WorkingDirectory
+        {
+            get
+            {
+                return _scriptBlockInitialLocation;
+            }
```

Comment:
I don't think the explicit backing field here is necessary. You can define this like so:

```csharp
public virtual string WorkingDirectory { get; set; }
```

and then just reference `WorkingDirectory` instead of the private field.